src/tests/test_helpers.cc

```diff
@@ -143,6 +145,7 @@ void PrintNoTouchPrompt() {
             << "not touch your security key until the test finishes. You\n"
             << "should see a flashing LED on the device, please ignore it.\n"
             << "===========================================================\n";
+  absl::SleepFor(absl::Milliseconds(2000));
```

Comment:
Why not `absl::Seconds(2)`?

Reply:
Because I used Milliseconds somewhere else (since the spec measures everything in milliseconds). :) Changed to seconds.